app/helpers/collections_helper.rb

```diff
@@ -41,4 +41,19 @@ def nrt_badge(metadata)
       content_tag(:span, 'NRT', class: 'eui-badge nrt')
     end
   end
+
+  def edsc_url
+    config_services = Rails.configuration.services
+    if Rails.env.production?
+      config_services.dig('earthdata', 'ops', 'edsc_root')
+    elsif Rails.env.sit? || Rails.env.development? || Rails.env.test?
```

Comment:
Is it possible to run a local EDSC?

Reply:
I dont think so (just trying https://search.localtest.earthdata.nasa.gov/search doesn't work) but @ryanmiller-1 @charleshuang80 may have insight.

Reply:
EDSC has instructions in their repo to run locally.  I don't think that this work warrants the extra complication to development and testing.

Reply:
If you edit your local services.yml, you can just point that location to a local EDSC I believe.